targets/ar71xx-mikrotik

```diff
@@ -1,9 +1,16 @@
 config 'CONFIG_GLUON_SPECIALIZE_KERNEL=y'
 
+ATH10K_PACKAGES='kmod-ath10k-ct ath10k-firmware-qca9887 ath10k-firmware-qca988x'
+if [ "$GLUON_WLAN_MESH" = 'ibss' ]; then
+	ATH10K_PACKAGES='kmod-ath10k-ct ath10k-firmware-qca9887-ct ath10k-firmware-qca988x-ct'
+fi
+
 # Enable ath5k in addition to ath9k
 # ath5k cards are commonly used with Mikrotik hardware
 packages 'kmod-ath5k'
 
+packages $ATH10K_PACKAGES
```

Comment:
Please only add the package assignment below a device that actually needs it, not globally for the target, as right now the packages will also be part of the devices that don't make use of it.

Reply:
i think this could be difficult as you can exchange the wifi modules in some mikrotik devices (like in other devices like x86 APU2 or mvebu turris omnia)

Reply:
Okay.